app/models/skill.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Skill < ApplicationRecord
+  has_many :user_skills
+  has_many :users, through: :user_skills
+
+  validates :name, presence: true, length: { within: 1..25 }
+
+  def to_s
+    name
+  end
+
+  def self.get_skills
+    Skill.all
+  end
```

Comment:
What is this for? It is redundant. `Skill.all` is already provided, no need to wrap it in your function.